incubator/tensorflow-inception/templates/inception-job.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
the problem with pre/post-install hooks is that the resources created in those hooks are not deleted with `helm delete release` and also the helm notes are not shown until the Job has finished.

As you said the init-container will make scale the server odd. 
I think it is fine to launch the Job at the same time as the other resources as the Deployment has readiness and liveness probes and it will be restarting pods a couple of times until the Job finishes.